### DISCUSSION TITLE
check if a resource exists

### DISCUSSION THREAD

**Discussion by dulong41** [argoproj/argo-workflows#11298] [Q&A] [answered]
I want to check whether a resource is existed with argo workflow, and I do not want to use kubectl image or other.
if you have native interface like using resource template or other methods?

**Comment by agilgur5** [comment, marked as answer]
Yes, you can use a [Resource Template](https://argoproj.github.io/argo-workflows/walk-through/kubernetes-resources/) with `action: get` to achieve this.
